.github/workflows/linting.yml

```diff
@@ -78,5 +78,3 @@ jobs:
             lint_log.txt
             lint_results.md
             PR_number.txt
-
-#
```

Comment:
I think this is from prettier

Reply:
Hmm not sure, but if I update this then it complains that it does not match the template in the linting tests, so I'm just gonna leave this in for now...

Reply:
Ah I think Phil fixed it: https://github.com/nf-core/tools/pull/1508